test/Templates.Test/Helpers/TemplateTestBase.cs

```diff
@@ -132,7 +132,13 @@ protected void RunDotNet(string arguments)
 
         protected void RunDotNetEfCreateMigration(string migrationName)
         {
-            var args = $"ef migrations add {migrationName}";
+            var assembly = typeof(TemplateTestBase).Assembly;
+
+            var DotNetEfFullPath = assembly.GetCustomAttributes<AssemblyMetadataAttribute>()
+                .FirstOrDefault(attribute => attribute.Key == "DotNetEfFullPath")
+                .Value;
+
+            var args = $"{DotNetEfFullPath} migrations add {migrationName}";
```

Comment:
Add quotes around the path just in case this is running on a machine with a space in the folder name.